packages/ui/lib/components/Select/Select.tsx

```diff
@@ -264,6 +276,17 @@ export const Select = <T extends unknown>({
                     />
                   </Listbox.Option>
                 )}
+                {!isOtherSelected && options.length > showMoreAfter && (
+                  <button
+                    className="w-full"
+                    type="button"
+                    onClick={() => setIsOtherSelected(true)}
+                  >
+                    <SelectOption
+                      option={{ label: 'Show more', value: CUSTOM_VALUE }}
```

Comment:
Let's see if we can use a different style for this option, like _italics_ and add `...`. So smething like _Show more..._.

Thanks!